README.md

```diff
@@ -11,7 +11,7 @@ Our goal is to provide a secure and reproducible way to build packages and conta
 
 - 🐳 No additional tools are needed except for [Docker](https://docs.docker.com/engine/install/)!
 - 🚀 Easy to use declarative configuration that provides reproducible builds
-- 📦 Build packages and/or containers for [Azure Linux](https://github.com/microsoft/azurelinux) 2 and 3 (formerly known as CBL-Mariner)
+- 📦 Build packages and/or containers for a number of different [targets](https://github.com/azure.github.io/dalec/targets)
```

Comment:
Users might not know what a target is, would we want to the operating systems? 

```suggestion
- 📦 Build packages and/or containers for a number of different [targets](https://github.com/azure.github.io/dalec/targets) such as Ubuntu and Azure Linux
```

Reply:
I like it, thanks!
Updated.